hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -138,6 +138,12 @@
       .sinceVersion("0.10.0")
       .withDocumentation("When enabled, populates all meta fields. When disabled, no meta fields are populated.");
 
+  public static final ConfigProperty<Boolean> VALIDATE_METADATA_PAYLOAD_CONSISTENCY = ConfigProperty
+      .key("_" + METADATA_PREFIX + ".validate.metadata.payload.consistency")
+      .defaultValue(false)
+      .sinceVersion("0.10.10")
+      .withDocumentation("Whether to perform validations on metadata payload consistency.");
```

Comment:
this needs to be bit more descriptive

Reply:
can we can this something like `.ignore.spurious.deletes` which is what it is. i.e deleting something that did not exist in the first place. validatexxxx makes it sound like something essential being turned off